### PR TITLE
Feat/language switcher

### PR DESCRIPTION
**Pull request by fredericvilcot**
This PR implements several issues related to internationalization:

- #65 that adds a language switcher component in the lateral bar
- #81 that removes deprecated `My Data Spaces` route in lateral bar navigation
- #59 that adds `FR` and `DE` translations for the whole lateral bar and the work in progress page
- #60 that adds `FR` and `DE` translations for the toolbar component

### CODE REVIEW

**Review by lolottetheclash — APPROVED** [review]
This is great !👌
Don't know if it's relevant but would it be interesting to have the languages menu sorted alphabetically? but perhaps it is not in the spec

File: src/hook/useClickOutside.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  useEffect(() => {
+    document.addEventListener('mousedown', listener)
+    document.addEventListener('touchstart', listener)
+
+    return () => {
+      document.removeEventListener('mousedown', listener)
+      document.removeEventListener('touchstart', listener)
+    }
+  }, [listener, ref])
+}
```
[the remark this answers is not in the document]
Comment (fredericvilcot) [reply src/hook/useClickOutside.ts]: @ccamel 
Great, I committed your changes 😌

File: src/component/sidebar/languageSwitcher/languageSwitcher.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+  const handleMenuItemClick = useCallback(
+    (lng: Lng) => () => {
+      changeLanguage(lng)
+      closeMenu()
+    },
+    [closeMenu]
+  )
+
+  const clickOutsidehandler = useCallback(() => {
```
[the remark this answers is not in the document]
Comment (fredericvilcot) [reply src/component/sidebar/languageSwitcher/languageSwitcher.tsx:45]: Well catch 👀

File: src/languages/languages.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    lng: 'fr',
+    label: 'français'
+  },
+  {
+    lng: 'de',
+    label: 'deutsch'
+  }
+]
+
+export const getActiveLanguage = (): Language | null =>
```
[the remark this answers is not in the document]
Comment (fredericvilcot) [reply src/languages/languages.ts]: @ccamel 
I made the mods using `fp-ts` and `Option` monad, I let you have a look 😌

**Review by ErikssonJoakim — APPROVED** [review]
Great work!

**Review by ccamel — APPROVED** [review]

**Pull request merged**